### PR TITLE
Issue/156/cleanup

### PR DESCRIPTION
**Pull request by eacharles**
These are just changes I made to get pylint to stop complaining.   

Maybe have a look to see what they were, but don't spend too much time reviewing all the details.

### CODE REVIEW

File: src/qp/quant_pdf.py [hunk 1]
```diff
@@ -193,12 +193,13 @@ def _instantiate_pdf_constructor(self):
             self._quants, self._locs)
 
 
-    def _pdf(self, x, row):
+    def _pdf(self, x, *args):
```
Comment (drewoldag) [inline src/qp/quant_pdf.py:196]: What was the pylint rule that it was complaining about here?

File: src/qp/sparse_pdf.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ def get_allocation_kwds(cls, npdf, **kwargs):
         if 'dims' not in kwargs:
             raise ValueError("required argument dims not in kwargs") #pragma: no cover
         nsp = np.array(kwargs['dims']).flatten()[4]
-        nmu = np.array(kwargs['dims']).flatten()[0]
+        # nmu = np.array(kwargs['dims']).flatten()[0]
```
Comment (drewoldag) [inline src/qp/sparse_pdf.py:80]: Can we just remove this line?

**Review by drewoldag — APPROVED** [review]
Looks good. Left two small comments, but nothing that's blocking the merge.

File: src/qp/quant_pdf.py (hunk 1 above)
Comment (eacharles) [reply src/qp/quant_pdf.py:196]: Signature didn't match base class.

File: src/qp/sparse_pdf.py (hunk 2 above)
Comment (eacharles) [reply src/qp/sparse_pdf.py:80]: I think so.

**Comment by eacharles** [comment]
Oops, I rebased the other pull request into this one.   Ignore that for now.  I'll merge the other once first.

**Review by OliviaLynn — APPROVED** [review]

**Comment by eacharles** [comment]
Ok, this got merged into the other PR.   Gonna close this.

**Pull request closed without merging**